README.md

```diff
@@ -100,4 +100,12 @@ You can also use tools like certmanager's [trust operator](https://cert-manager.
 ## Development
 `make build`
 
-`docker build -f package/Dockerfile . -t $MY_REPO:$MY_TAG`
\ No newline at end of file
+`docker build -f package/Dockerfile . -t $MY_REPO:$MY_TAG`
+
+## Release
+
+1. Update the [rancher-csp-adapter chart](https://github.com/rancher/charts/blob/dev-v2.6/packages/rancher-csp-adapter/package.yaml).

```

Comment:
Might be better for this to say something like:

```markdown 
1. Clone the repo (not your fork). 
2. Pull the `release/v2.6` branch
3. Tag the branch with your new version. For example, to release version 1.1.2, tag with `git tag v1.1.2` (make sure to include the v, it's important for the CI).
4. Push the new tag to the remote: `git push --tags`. Wait for the pipeline to release the new version before moving on to the next step.
5. In the charts repo, update the version of the adapter. See the [charts](https://github.com/rancher/charts/blob/dev-v2.6) repo for instructions on updating the [rancher-csp-adapter chart](https://github.com/rancher/charts/blob/dev-v2.6/packages/rancher-csp-adapter/package.yaml)
```

This way we have a comprehensive guide on each step. Goal here should be that someone who has never released the adapter before can do so without asking any of us questions.

Reply:
Good point, added the sub-instructions.